### PR TITLE
Change PathFinder#get to get()

### LINKED ISSUE #36: Change PathFinder's get to get()
[`PathFinder#get`](https://github.com/sbt/io/blob/v1.0.0-M9/io/src/main/scala/sbt/io/Path.scala#L175) is defined as `def get: Seq[File]`, but its behaviour is to look at the filesystem.

Therefore I argue it should be `def get()`.

With regards to source compatibility fortunately going in this direction is safe.

### PR DESCRIPTION
**Pull request by dwijnand**
Given it side-effects by looking at the FS it should have parens.

With regards to source compatibility fortunately going in this direction
is safe.

Fixes #36

### CODE REVIEW

**Comment by typesafe-tools** [comment]
To validate this pull request against other sbt modules, please visit <A HREF="https://jenkins.scala-sbt.org:8592/job/sbt-validator/parambuild/?io=pull/104/head&deploy_to_bintray=false&run_sbt_community=false">this link.</A>

File: io/src/main/scala/sbt/io/Path.scala [hunk 1]
```diff
@@ -375,7 +375,7 @@ sealed abstract class PathFinder {
    * The files returned by this method will reflect the underlying filesystem at the time of calling.
    * If the filesystem changes, two calls to this method might be different.
    */
-  final def get: Seq[File] = {
+  final def get(): Seq[File] = {
```
Comment (eed3si9n) [inline io/src/main/scala/sbt/io/Path.scala:377]: Are we adding empty parentheses because it forces evaluation?

Comment (eed3si9n) [reply io/src/main/scala/sbt/io/Path.scala:377]: Would this logic apply to `getURLs`?

Comment (dwijnand) [reply io/src/main/scala/sbt/io/Path.scala:377]: Yes to both. I'll look at the uses of get and see if there are more to add.

**Review by eed3si9n — APPROVED** [review]

**Pull request merged**